client/index.html

```diff
@@ -356,25 +357,352 @@ <h1 class="title">Settings</h1>
+								<code>/query nick</code>
+							</div>
+							<div class="description">
+								<p>Send a private message to the specified user.</p>
+							</div>
+						</div>
+
+						<div class="help-item">
+							<div class="subject">
+								<code>/quit [message]</code>
+							</div>
+							<div class="description">
+								<p>
+									Disconnect from the current network with an optional message.
+								</p>
+							</div>
+						</div>
+
+						<div class="help-item">
+							<div class="subject">
+								<code>/raw message</code>
+							</div>
+							<div class="description">
+								<p>Send a raw message to the current IRC network.</p>
+								<p>Aliases: <code>/quote</code>, <code>/send</code></p>
+							</div>
+						</div>
+
+						<div class="help-item">
+							<div class="subject">
+								<code>/slap nick</code>
+							</div>
+							<div class="description">
+								<p>Slap someone in the current channel with a trout!</p>
+							</div>
+						</div>
+
+						<div class="help-item">
+							<div class="subject">
+								<code>/topic newtopic</code>
+							</div>
+							<div class="description">
+								<p>Set the topic in the current channel.</p>
+							</div>
+						</div>
+
+						<div class="help-item">
+							<div class="subject">
+								<code>/voice nick [...nick]</code>
+							</div>
+							<div class="description">
+								<p>
+									Give voice (<code>+v</code>) to one or several users in the
+									current channel.
 								</p>
 							</div>
 						</div>
+
+						<div class="help-item">
+							<div class="subject">
+								<code>/whois nick</code>
+							</div>
+							<div class="description">
+								<p>
+									Retrieve information about the given user on the current
+									network.
+								</p>
+							</div>
+						</div>
+
+						<h2>Keyboard Shortcuts</h2>
+
+						<h3>On Windows / Linux</h3>
+
+						<div class="help-item">
+							<div class="subject">
+								<kbd>Ctrl</kbd> + <kbd>↑</kbd> / <kbd>↓</kbd>
+							</div>
+							<div class="description">
+								<p>Switch to the previous/next window in the channel list</p>
+							</div>
+						</div>
+
+						<div class="help-item">
+							<div class="subject">
+								<kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>L</kbd>
+							</div>
+							<div class="description">
+								<p>Clear the current screen</p>
+							</div>
+						</div>
+
+						<h3>On macOS</h3>
+
+						<div class="help-item">
+							<div class="subject">
+								<kbd>⌘</kbd> + <kbd>↑</kbd> / <kbd>↓</kbd>
+							</div>
+							<div class="description">
+								<p>Switch to the previous/next window in the channel list</p>
+							</div>
+						</div>
+
+						<div class="help-item">
+							<div class="subject">
+								<kbd>⌘</kbd> + <kbd>K</kbd>
+							</div>
+							<div class="description">
+								<p>Clear the current screen</p>
+							</div>
+						</div>
+
+						<h2>About The Lounge</h2>
+
+						<p class="about">
```

Comment:
Honestly, this feels a bit like it's being shoved in here. I'm not sure about this. It feels like a separate thing, to be honest. I know you want this to be an all encompassing help page, but this doesn't really feel like it fits.